packages/server/src/utils/index.ts

```diff
@@ -328,6 +328,14 @@ export const buildLangchain = async (
                 logger.debug(`[server]: Finished upserting ${reactFlowNode.data.label} (${reactFlowNode.data.id})`)
                 break
             } else {
+                // skip initializing web scraper nodes during prediction since they would have already run as a part of upsert
```

Comment:
@0xi4o actually ignore me, my bad i was testing on diff branch. The addition of this make it not working. I think its fine as it is. If you can just remove this

Reply:
also if u can pull latest changes from main

Reply:
Reverted the changes.

Reply:
Pulled from main as well.